### ISSUE TITLE
Consider XML as valid UI file

### ISSUE THREAD

**Issue by jzeferino** [jmmortega/Measurer4000#8]
When I write Android UI's typiclly I use `.xml` insteand of `.axml`.

In the `WellSmellsLikeFile` should be considered the `.xml` as a valid UI file.

https://github.com/jmmortega/Measurer4000/blob/63fe638191e6d4ee403f91f160bbf30112ccf49b/Measurer4000.core/Utils/ProjectIdentificatorUtils.cs#L141

Can I fix that? Do you see any issues changing it?

**Comment by jmmortega** [comment]
@jzeferino In fact, I don't add .xml because in this case include localization files, drawable files, etc.

So, in this case maybe it's preferable include a checkbox or something to set this setting, what do you think?

**Comment by jzeferino** [comment]
Since Android layouts could be a .xml file, and drawables are xml too and considered layout IMO.
This could be considered by default but we must discar the Resource folder with the strings values witch is a xml too.

From what i've seen in the code there ins't a google place to filter this xml files only for android, what do you suggest?

**Comment by xleon** [comment]
I don´t see any reason to discard localization files, Android Resource files and friends from the stats. That´s a part of the project like any other code file right?

**Comment by jzeferino** [comment]
Yes it could be part of the metrics but not only in android, ios string files must be taken in account too.

**Comment by jmmortega** [comment]
But, I think localization files isn't a maintanable file per se. Maybe included a new section call "Localization lines" or something like that. It's a good idea, what do you think?
